content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -116,7 +117,6 @@ different Kubernetes components.
 | `ProcMountType` | `false` | Alpha | 1.12 | |
 | `QOSReserved` | `false` | Alpha | 1.11 | |
 | `RemainingItemCount` | `false` | Alpha | 1.15 | |
-| `RequestManagement` | `false` | Alpha | 1.15 | |
```

Comment:
some thoughts on this: if `RequestManagement` has been the name of the feature gate since 1.15 and it is renamed to `APIPriorityAndFairness` starting 1.17, we should faithfully record this. Instead of removing line 119, we should put

```
| `RequestManagement` | `false` | Alpha | 1.15 | 1.16 |
```

And for line 53, we should put:

```
| `APIPriorityAndFairness` | `false` | Alpha | 1.17 |  |
```

Correct?

Reply:
@tengqm This is good for me too.
So I wrote `RequestManagement` into the deprecated table and add "Renamed from RequestManagement" into the `APIPriorityAndFairness` description.